backend/app/views/spree/admin/shared/_store_switcher.html.erb

```diff
@@ -0,0 +1,17 @@
+<% if can?(:admin, current_store) %>
+  <li class="d-block internationalization-options">
+    <div class="dropdown border-top">
+      <button id="internationalization-button-desktop" type="button" data-toggle="dropdown" class="dropdown-toggle p-3" aria-label="<%= Spree.t(:internationalization) %>">
+        <%= current_store.name %>
+      </button>
+      <div id="internationalization-options-desktop" class="dropdown-menu dropdown-menu-right text-left">
```

Comment:
@Vegann simple `<select>` element will do, we don't need additional information here